src/Collector/TemplatePathCollector.php

```diff
@@ -74,8 +74,8 @@ public function processNode(Node $node, Scope $scope): ?array
         }
 
         /** @var string|null $path */
-        $path = $this->valueResolver->resolve($arg->value, $scope->getFile());
-        if ($path === null) {
+        $path = $this->valueResolver->resolvePath($arg->value, $scope->getFile());
+        if ($path === null || $path[0] !== '/') {
```

Comment:
$path[0] is not set for empty string

Reply:
empty string cannot be returned here it is covered by resolvePath